cocotb/regression.py

```diff
@@ -458,7 +458,19 @@ def _my_test(dut):
 
 
 class TestFactory(object):
-    """Used to automatically generate tests.
+    """Factory to automatically generate tests.
+
+    Args:
+        test_function: The function that executes a test.
+            Must take *dut* as the first argument.
+        *args: Remaining arguments are passed directly to the test function.
+            Note that these arguments are not varied. An argument that
+            varies with each test must be a keyword argument to the
+            test function.
+        *kwargs: Remaining keyword arguments are passed directly to the test function.
```

Comment:
```suggestion
        **kwargs: Remaining keyword arguments are passed directly to the test function.
```

It's possible for this to render correctly with sphinx, that you need to use `\*\*` and use an `r"""` string

Reply:
Good catch, fixed.
Sphinx appears to handle the unescaped ``**kwargs`` just fine though, have you seen otherwise?